test/upgrade.js

```diff
@@ -58,6 +70,12 @@ function run(t, fixture, opts) {
 , [ 'upgrades code with define wrapper function definition after wrapper'                  , 'define-function-after' ]
 , [ 'upgrades code with define wrapper that returns a multiline object'                    , 'define-return-multiline-object' ]
 , [ 'upgrades code with require wrapper that has dependencies but no params'               , 'require-noparams' ]
+, [ 'upgrades code with use strict statement at top of file'                               , 'use-strict', moveStrictTrueOpts ]
+, [ 'upgrades code with use strict statement without moving'                               , 'use-strict-not-moved', moveStrictFalseOpts ]
+, [ 'upgrades code with use strict statement in nested function'                           , 'use-strict-nested-not-moved', moveStrictTrueOpts ]
+, [ 'upgrades code with use strict in the middle of the module'                            , 'use-strict-in-middle', moveStrictTrueOpts ]
+, [ 'upgrades code with use strict at end of the module'                                   , 'use-strict-at-bottom', moveStrictTrueOpts ]
+, [ 'upgrades code with use strict without semicolon'                                      , 'use-strict-without-semicolon', moveStrictTrueOpts ]
```

Comment:
This last is is a bit weird, as semicolons are inserted after transformation.

It's done so on both the `require`s and `module.exports`' though, so I'm thinking that can be a separate option another time? It correctly moves the `use strict` now, with or without `;`, which is more important I think.